src/components/CreateComment.jsx

```diff
@@ -0,0 +1,43 @@
+import { useEffect, useState } from "react";
+import { postComment } from "../../utils/api";
+
+function CreateComment({ article_id, setCommentsList }) {
+
+    const [body, setBody] = useState("")
+    const [username, setUsername] = useState('grumpy19')
+    const [hasCommented, setHasCommented] = useState(false)
+
+    function handleSubmit(event) {
+        
+        event.preventDefault();
+        const commentToPost = {
+            author: username,
+            body: body,
+            votes: 0
+        }
+
+        setCommentsList((currComments) => [commentToPost, ...currComments])
+        setBody('')
+        setHasCommented(true)
+        postComment(article_id, commentToPost)
+
```

Comment:
what if the api call fails, you will need to remove the comment from the array in state and inform the user that their comment wasn't posted?